x-pack/plugins/lens/public/xy_visualization/xy_config_panel/shared/line_style_settings.tsx

```diff
@@ -40,7 +40,7 @@ export const LineStyleSettings = ({
           defaultMessage: 'Line',
         })}
       >
-        <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween">
+        <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween" responsive={false}>
           <EuiFlexItem grow={true}>
```

Comment:
I believe the `grow={true}` prop can be removed here, as this is the default for the component.

```suggestion
          <EuiFlexItem>
```